lib/generators/react_on_rails/templates/lib/tasks/linters.rake

```diff
@@ -0,0 +1,71 @@
+if %w(development test).include? Rails.env
+  namespace :lint do
+    # require "rubocop/rake_task"
+    # require "slim_lint/rake_task"
+    require "scss_lint/rake_task"
```

Comment:
if the gem scss_lint is not in the Gemfile, running "rake -T" is going to fail here. 

I could adjust it to be flexible about the scss_lint being installed with this code:

```ruby

  desc "See docs for task 'scss_lint'"
    task :scss do
      begin
        require 'scss_lint/rake_task'
        SCSSLint::RakeTask.new do |t|
          t.files = ["app/assets/stylesheets/", "client/assets/stylesheets/"]
        end
        Rake::Task[:scss_lint].invoke
      rescue LoadError
        puts "** add gem 'scss_lint' to your Gemfile for scss linting."
      end
    end
```